docker/Dockerfile.ubuntu.xenial

```diff
@@ -0,0 +1,35 @@
+# Dockerfile for building an image with ubuntu xenial
+FROM sociomantictsunami/base
+MAINTAINER andreas-bok-sociomantic <andreas.bok@sociomantic.com>
+
+WORKDIR /root
```

Comment:
:scream:

Reply:
hehe, will change to a less contested dir

Reply:
Afterwards you change it to `libdrizzle-redux` or something. I wonder why you set it at all.